pkg/cli/query.go

```diff
@@ -33,24 +33,23 @@ func Query() *cobra.Command {
 		Example: `  melange query config.yaml "{{ .Package.Name }}-{{ .Package.Version }}-{{ .Package.Epoch }}"`,
 		Args:    cobra.MinimumNArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return QueryCmd(cmd.Context(), args[1], build.WithConfig(args[0]))
+			return QueryCmd(cmd.Context(), args[0], args[1])
 		},
 	}
 
 	return cmd
 }
 
-func QueryCmd(ctx context.Context, pattern string, opts ...build.Option) error {
-
-	bc, err := build.New(opts...)
+func QueryCmd(ctx context.Context, configFile, pattern string) error {
```

Comment:
This method doesn't need to be exported.

Reply:
True, but every other command in `cli/*.go` is, so I just followed the pattern. If we want to make those private, we should do it across the board.